My capstone project

diarisdiakite opened the pull request:
In this PR, I:
- Created the project
- Added Linters files
- personalized the content of the 2 pages (Home and about pages) with a topic different from the one in the original design.
- followed the [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), including:
  - Colors.
  - Typography: font face, size and weight (we suggest using [Lato](https://www.latofonts.com/) as it is a free font similar to the one used in the original design).
  - Layout: composition and space between elements.
- Used the CSS framework _Bootstrap_ for styling
- Added a 3rd page: Join page*.
- Created versions for 2 different screen sizes for each of the pages; Mobile: up to 768px wide, Desktop: 768px or wider.

- For Interactions
  - The *home page* has a link in the menu to the *about page*;
  - The logo in the header links to the *home page*;
  - The Mobile menu appears over the page when the user clicks (or taps) the hamburger button on the header;
  - For dynamic pages: the "Speakers" and "Participants" sections have been created dynamically in JavaScript, as well as the "navbar", "footer", and "program" sections. The "Speakers" and "Participants" sections were created from a "list.js" file added as a module.
 
- Added the credit Original design idea by [Cindy Shin in Behance](https://www.behance.net/adagio07).

diarisdiakite committed:
Change title

diarisdiakite committed:
Chance linter.yml

diarisdiakite committed:
Chance linter.yml

diarisdiakite committed:
Add linters files

diarisdiakite committed:
Add Readme.md and cocogoose font

diarisdiakite committed:
Add images

diarisdiakite committed:
Update index.html. Add bootstrap containers for different sections

diarisdiakite committed:
Add colors variables in css root for main colors. Add css classes

diarisdiakite committed:
Add navbar and footer with javascript files.

diarisdiakite committed:
Add about.html page

diarisdiakite committed:
Add speakers list to be extended to main the exhibition hall partipants

diarisdiakite committed:
Add speakers with Javascript

diarisdiakite committed:
Add a join page

diarisdiakite committed:
Add participants list and arrow for the speakers collapsed div

diarisdiakite committed:
Fix speakers display for Mobile version.

diarisdiakite committed:
Complete mobile version. Fixing desktop version.

diarisdiakite committed:
Add a program.js and an animation files

diarisdiakite committed:
Update images. Fix image sizing. Add program.js related classes.

diarisdiakite committed:
Add participants list

diarisdiakite committed:
Add participants class content through javascript.

diarisdiakite committed:
Add navbar social icons

diarisdiakite committed:
Updat index.html page with javascript content. Add image opacity to about.html. Update form on join.html.

diarisdiakite committed:
Update Readme with the video and the credit.

diarisdiakite committed:
Fix linters issues

diarisdiakite committed:
Delete hint report file

Qoosim commented on index.html:
```diff
@@ -1,13 +1,183 @@
+      <div class="carousel-inner">
+        <div class="carousel-item w-100 active">
+          <img class="w-100 h-50 img-fluid" src="assets/img/carousel/3.jpg" alt="">
+          <div class="carousel-caption">
```
- [x] Wonderful job. However, the point here is to trained how to follow client specification and that should be done on this project. follow the design to the letter. What is only required is to persionalize the content of the pages involve. Every other things should remain. This includes color, typography, font, background-color among others. Kindly stick to the design as close as possible.

Qoosim commented on index.html:
```diff
@@ -1,13 +1,183 @@
+        <div class="carousel-item w-100 active">
+          <img class="w-100 h-50 img-fluid" src="assets/img/carousel/3.jpg" alt="">
+          <div class="carousel-caption">
+            <h1 class="display-4 text-left">ANNUAL SMSE SUMMIT</h1>
+            <h2 class="display-4 text-left">2023.11.24(Fri) to 26(Sun)</h2>
+            <h3 class="display-5 text-left">@International Center of Conference - Bamako(Mali)</h3>
+          </div>
+        </div>
+        <!-- -->
+        <div class="carousel-item w-100">
+          <img class="w-100 img-fluid" src="assets/img/carousel/2.jpg" alt="">
+          <div class="carousel-caption">
+            <div class="col-12">
+              <h2 class="display-5 my-text-red text-center lead bold">Theme: SMSE Digitalization challenges in developing countries</h2>
+              <p class="my-bg-light my-text-red bold lead p-4"> <!-- trying different classes //class="lead"-->
+                A worldwide competition between Small and Medium Size Entreprises.
+                with an Exhibition hall of 100 entreprises and high level speakers.
+                <br> 
+                Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi 
+                ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit 
+                in voluptate velit esse cillum dolore eu fugiat nulla pariatur. 
+                Excepteur sint occaecat cupidatat non proident, 
+                sunt in culpa qui officia deserunt mollit anim id est laborum.
+                <br>
+              </p>
+            <div class="mb-4">
+              <span class="badge my-bg-red p-3">Tech entreprises</span>
+              <span class="badge my-bg-red p-3">Craft industry</span>
+              <span class="badge my-bg-red p-3">Construction</span>
+              <span class="badge my-bg-red p-3">Food processing</span>
+            </div>
+            <h2>2023.11.24(Fri) to 26(Sun)</h2>
+            <h3>International Center of Conference - Bamako(Mali)</h3>
+          </div>
+        </div>
+        <!-- -->
+      </div>
+    </div>
+    </div>
```
- [x] It is required to produce a close replica of the guiding design but with the persionalized content. Kindly stick to this requirement.

### Project Requirements
![project-rqt](https://user-images.githubusercontent.com/26597432/234996627-0127375f-f681-4223-9ba7-824dbb266e4f.png)

### Your Design
![your-top-section](https://user-images.githubusercontent.com/26597432/234996671-1facb9a6-83de-4cea-81de-bbe47f836949.png)

### Guiding Template
![temp-top-section](https://user-images.githubusercontent.com/26597432/234996708-7223c28e-078c-42ab-9d82-e0308172ede5.png)

diarisdiakite replied:
Done @Qoosim

diarisdiakite replied:
@Fixed @Qoosim

Qoosim commented on index.html:
```diff
@@ -1,13 +1,183 @@
+    </div>
+    </div>
+      <!-- MAIN PROGRAM-->
+    <div class="container-fluid my-bg-gray py-4" id="main-program">
+      <h3 class="text-center">
+        Main program
+      </h3>
+      <hr class="full-width mx-auto mb-5">
+      <div class="container">
+        <div id="program-cards" class="row text-center">
+          <!-- TO BE ADDED DYNAMICALLY-->
+
+
+        
+        </div>
+        
+      <!-- TO BE ADDED DYNAMICALLY-->
+      </div>
+    </div>
```
- [x] Same thing apply here too. Kindly produce the exact design from the guiding template, but with persionalized content. The number of cards are five on the guding tempate while yours are six.
- [x] Each card has background color while your cards have no background color. 
- [x] You must also work on the width, padding, and margin of each card to be the same as we have them on the guiding template.

### Your Design
![your-main-program](https://user-images.githubusercontent.com/26597432/234997564-d074bb84-241a-4016-93dd-c0c3a9c04afa.png)

### Guiding Template
![temp-main-program](https://user-images.githubusercontent.com/26597432/234997688-3ca7f136-b833-4ac6-9e34-ef7281c1648d.png)

### Mobile Review

- [x] Your layout for the main program section on mobile does not look like how the section was layed out on the mobile. Kindly work your design alongside the template layout.

### Your Design
![your-main-program-mobile](https://user-images.githubusercontent.com/26597432/235000933-2566a8b1-b15c-4b3a-8be7-488edca68a61.png)

### Template
![temp-main-program-mobile](https://user-images.githubusercontent.com/26597432/235000967-38a72adb-8597-4cda-a20e-74dc4b54febe.png)

diarisdiakite replied:
Well received @Qoosim.
The program section has been added through Javascript. It has been updated to have backgound.
Please would you refresh the page when passing from one layout(screen size) to the other one?

Qoosim commented on index.html:
```diff
@@ -1,13 +1,183 @@
+  <title>Annual SMSE Summit welcome page</title>
 </head>
 <body>
+  <header>
+    <!--ADD THE FIRST LEVEL -->
+    <div class="container-fluid display-desktop social" id="social">
+      
+    </div>
+    <!--ADD THE NAVBAR -->
+    <nav class="navbar navbar-expand-md navbar-light bg-light sticky-top" id="my-navbar">
+      
+    </nav>
+  </header>
```
- [x] The height of the dark part of your navbar is higher compare to how it is on the template. Kindly check the height and work your design alongside.
- [x] The text color on your dark top navbar is not the same as the color on the original design. Kindly fix that too.

### Your Design
![your-navbar](https://user-images.githubusercontent.com/26597432/234998838-6dc42fef-6264-4919-b353-14f424317c64.png)

### Guiding Template
![temp-navbar](https://user-images.githubusercontent.com/26597432/234998860-0d69b96d-a649-45e3-be27-5620faae442f.png)

diarisdiakite replied:
Done @Qoosim

Qoosim commented on index.html:
```diff
@@ -1,13 +1,183 @@
+      </div>
+    </div>
     
+    <!-- SPEAKERS -->
+    <div class="container w-75 py-4">
+      <h3 class="text-center">Speakers</h3>
+      <hr class="full-width mx-auto mb-5">
+      <div class="row speakers" id="speakers">
+      
+        
+      </div>
+    </div>
```
- [x] Your Design for `Feature speakers` on mobile deos not tally with the layout on the given template. Kindly work your design alongside the guiding template.

### Your Design
![your-feature-speaker-mobile](https://user-images.githubusercontent.com/26597432/235000143-3ad42be2-32af-4ac5-9adb-8da78c16e389.png)

### Guiding Template
![temp-feature-speakers-mobile](https://user-images.githubusercontent.com/26597432/235000192-2f4cc244-7887-48c4-9702-7bc6f6cde476.png)

diarisdiakite replied:
Fixed @Qoosim. Thank you for the comment.

KayLemba replied:
Please as the previous code reviewer suggested kindly fix this and please stick to design rules as much as possible to avoid making mistakes  kindly add the checked background like it has been provided to you in template.

This is your design:
![Screenshot 2023-04-29 at 3 13 19 AM](https://user-images.githubusercontent.com/79506105/235276799-615214af-e359-452f-8a2c-7629b89a67a1.png)

Please stick to this design: 

![Screenshot 2023-04-29 at 3 27 39 AM](https://user-images.githubusercontent.com/79506105/235276904-c630d17c-c9eb-4276-9084-0702a89a3d4c.png)

diarisdiakite replied:
Done @KayLemba. Thanks for the comment, didn't notice this one the first time.

Qoosim commented on about.html:
```diff
@@ -0,0 +1,178 @@
+  <title>Art Selection welcome page</title>
+</head>
+<body>
+  <header>
+    <!--ADD THE FIRST LEVEL -->
+    <div class="container-fluid display-desktop social" id="social">
+      
+    </div>
+    <!--ADD THE NAVBAR -->
+    <nav class="navbar navbar-expand-md navbar-light bg-light sticky-top" id="my-navbar">
+      
+    </nav>
+  </header>
```
- [x] Kindly apply the suggestion from home page on this page too.
- [x] There's no menu option to navigate to the home page from about page on the mobile version like your did on the home page when the hambuger icon is clicked. Kindly look into that and fix.

### Screen Recording
[your-about-to-home-missing.webm](https://user-images.githubusercontent.com/26597432/235002856-c861a64f-9e24-4c1c-965a-ad0290f0d7f3.webm)

diarisdiakite replied:
Done @Qoosim

Qoosim commented on about.html:
```diff
@@ -0,0 +1,178 @@
+          </div>
+          <div class="row text-center space">
+  
+            <div class="col-xs-12 col-sm-6 col-md-4 item1">
+              <img class="img-thumbnail img-fluid myMask" id="past-summits" src="assets/img/past-summits/2.jpg" alt="">
+              
+              <div class="past-summits-img-caption text-center card-img-overlay">
+                <h3 class="display-6 bold">2020</h3>
+                <h6 class="display-5 bold">ANNUAL SMSE SUMMIT 2020</h6>
+              </div>
+            </div>
+  
+            
+            <div class="col-xs-12 col-sm-6 col-md-4 item2">
+              <img class="img-thumbnail img-fluid myMask" id="past-summits" src="assets/img/past-summits/3.jpg" alt="">
+              <div class="past-summits-img-caption text-center card-img-overlay">
+                <h3 class="display-6 bold">2021</h3>
+                <h6 class="display-5 bold">ANNUAL SMSE SUMMIT 2021</h6>
+              </div>
+            </div>
+  
+            <div class="col-md-4 item3">
+              <img class="img-thumbnail img-fluid position-relative myMask" id="past-summits-images" src="assets/img/past-summits/4.jpg" alt="">
+              <div class="past-summits-img-caption text-center card-img-overlay">
+                <h3 class="display-6 bold">2022</h3>
+                <h6 class="display-5 bold">ANNUAL SMSE SUMMIT 2022</h6>
+              </div>
+            </div>        
+          </div>
+        </div>
+      </div> <!-- check if not an extra div -->
```
- [x] Kindly add only two pictures here to match what we have on the guiding template.

### Your Design
![your-pix-about-page](https://user-images.githubusercontent.com/26597432/235002158-30fc97b5-9959-4db8-86f5-58e940434b3e.png)

### Guiding Template
![temp-pix-about-page](https://user-images.githubusercontent.com/26597432/235002203-3f625289-6faa-4178-8ab7-bf4cf8d60869.png)

diarisdiakite replied:
Done @Qoosim, thank you for the comment

KayLemba replied:
Please kindly fix this as well and stick to design rules again kindly the back ground color of  your images to orange just as it is in the design follow design rules and stick to the template as much as possible.

diarisdiakite replied:
Done @KayLemba. Added and after pseudo-classe. Thanks for the reminder..

Qoosim commented on about.html:
```diff
@@ -0,0 +1,178 @@
+            <h1 class="display-4 mb-3">ANNUAL SMSE SUMMIT 2023</h1>
+            <hr class="full-width mx-auto mb-5">
+            <div class="col-12">
+              <p class="alert-info lead p-4"> <!-- trying different classes //class="lead"-->
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit, 
+                sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. 
+                Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi 
+                ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit 
+                in voluptate velit esse cillum dolore eu fugiat nulla pariatur. 
+                Excepteur sint occaecat cupidatat non proident, 
+                sunt in culpa qui officia deserunt mollit anim id est laborum.
```
- [x] It is required to stick to the guiding template as much as possbile.  Text color and the background color on your design does not tally with the text color and the background color on the original design.
- [x] You might also need to consider the width of the box.

### Your Design
![your-about-first-part](https://user-images.githubusercontent.com/26597432/235004007-cae37b29-f619-4e63-bcb1-5656f14129b9.png)

### Guiding Template
![temp-about-first-part](https://user-images.githubusercontent.com/26597432/235004025-abd97b1a-122b-4acb-9f8a-6381a70c1045.png)

diarisdiakite replied:
@Qoosim Boxes and texts fixed

Qoosim reviewed (changes requested):
#  *Status: Required Changes* :recycle:

Hi @diarisdiakite, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Fair README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 

![](https://camo.githubusercontent.com/38ce8c8b213b08e2d9bb85a0b79702ad512f3910f803ec3d7917b4e0110c9044/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f7741366159414f336a4836526d55494259732f67697068792e676966)


## Required Changes ♻️
_Check the comments under the review._

- [ ] It is required to record a video presenting the features of the project. 
   -  The video should contain a clear and objective description about the project in professional English. 
   -  The topic and purpose of the project should be made clear. 
   -  The video should contain a demo of the project, showing the pages in different screen sizes.
   - The video should contain a highlight about a piece of the code you find interesting or you're proud of making.
   - The video should have a maximum length of 5 minutes.
  Kindly do that.
   
### Project Requirement
![video-reqtm](https://user-images.githubusercontent.com/26597432/235005387-850548ce-ecec-4e74-8ee1-48997777343d.png)

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

diarisdiakite committed:
Fix speakers cards display on index.html. Delete one card from past events section and fixed boxes size on the about page.

diarisdiakite committed:
Integrate CR comments

diarisdiakite committed:
Fix linters ESlint errors

diarisdiakite commented:
In this milestone, I:
- Fixed the Boxes and texts display issues
- Availed the menu option to navigate to the home page from about page on the mobile version like I did on the home page 
- Reduced the height of the dark part of the navbar ito conform to the template.
- The text color on your dark top navbar is not the same as the color on the original design. Kindly fix that too.
- Redesigned the header, removing the slider.
- Added a new video instead of the former one at the demo link.

diarisdiakite committed:
Change the video link

KayLemba reviewed (changes requested):
Hi @diarisdiakite  👋
Good job so far working on the project requirements!
There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight 🎉
No linter errors. ✔️
GitHub flow is followed. ✔️

Required Changes ♻️
Check the comments under the review in the inline comments.

Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

diarisdiakite committed:
Integrate Code Reviewer comments

diarisdiakite committed:
Integrate Code Reviewer comments

diarisdiakite committed:
Integrate CR's comment

KayLemba commented on assets/js/speakers.js:
```diff
@@ -0,0 +1,51 @@
+
+const participantsContainer = document.getElementById('participantsJs');
+
+participants.forEach((participant) => {
```
[OPTIONAL]It would be nice to get rid of all commented code for it makes your codebase look untidy and it says in the project requirements that the speaker section should be generated using DOM manipulation.

diarisdiakite replied:
Fixed @KayLemba, thanks for the review.

diarisdiakite committed:
Add partners logos

KayLemba commented on index.html:
```diff
@@ -1,13 +1,172 @@
+    </div>
+
+
+
```
Please kindly fix the vertical scroll when i switch to mobile view and align page content to the center.

This is how your work looks:

![Screenshot 2023-04-29 at 5 55 47 AM](https://user-images.githubusercontent.com/79506105/235284387-88e0594e-2953-4037-bd82-09bdeb869584.png)
Please address this issue to avoid wasting review chances

diarisdiakite replied:
Fixed @KayLemba, thanks for the comment.

diarisdiakite committed:
Change past summits image caption color

KayLemba commented on index.html:
```diff
@@ -1,13 +1,172 @@
+        <div class="col-3 space" id="partner">
+          <a href=""><img class="img-thumbnail w-100 h-100" src="assets/img/logo/logo-partners/R.gif" alt=""></a>
+        </div>
+        <div class="col-3 space" id="partner">
```
The partners section should not be visible in the home page of the mobile screen. It would be nice it you fixed this issue.

This is your design:
![partners ](https://user-images.githubusercontent.com/79506105/235284500-955778a0-9982-4b3e-9c8b-2c3c0f2ef27e.png)


Please follow design requirements your design  should look like this in you home page:

![Screenshot 2023-04-29 at 7 08 21 AM](https://user-images.githubusercontent.com/79506105/235284570-d10f0fb2-d994-4f3b-a3c4-95edcc993f33.png)


Please Bare in mind that the partnes section should be visible in the about section of your mobile screen Just like in the design.

For a better understanding please follow the following requirements and guidelines:

Page Structure and Navigation

The project contains a Home page and About page.
It's possible to navigate between the pages, there's a link from Home page to About page and the opposite too.
Both pages contain the same navbar and footer.


Navbar

Mobile Size

It contains an hamburger menu on the left side.
It's background is transparent or the same color/pattern as the top section background.


Desktop Size

It is divided in a top dark bar and a bottom white bar.
The top short bar section with social media links, logout, and language links aligned to the right.
The taller bottom bar with a logo and a list of page links.
The top bar contains a logo to the left.
The top bar contains a list of page links aligned to the right.


Footer

It contains a logo aligned to the left and text aligned to the right.


Mobile Size

It has a white background.


Desktop Size

About Page: It has a dark background.
Home Page: It has a white background.


Home Page

It contains a top section with a column that includes a small line of text, the page title, a block of text, a medium line of text in bold letters, and a smaller line of text.
It contains a middle section with a column that includes a section header, a divider, five individual articles, each with an image, title, and description, and a line of text at the bottom.
It contains a bottom section with a column that includes a section header, a divider, and six individual articles, each with an image, title, subtitle, and description. Each article has two columns, the first with the image at the left and the second with the title, subtitle, and description.


Mobile Size

The middle section article list is organized in a vertical column.
The middle section individual articles have their elements organized in an horizontal row, with the image at the left, title in the center, and description at the right.
The bottom section articles are organized in a vertical column.


Desktop Size

The middle section article list is organized in an horizontal row.
The middle section individual articles have their elements organized in an vertical column, with the image at the top, title in the center, and description at the bottom.
The bottom section articles are organized in two columns of three articles each.


About Page

It contains a top section with a centered column that includes a page title, a text block with the page description, and a line with a contact information or any other prompt under it.
It contains a middle section with a centered column that includes a section header, a divider, a paragraph of text, and an image or large logo.
It contains a bottom section with a centered column that includes a section header, a divider, a line of text, and two images with text centered over them.


Mobile Size

The bottom section images are organized in a vertical column.


Desktop Size

The bottom section images are organized in an horizontal row.


Design and Styling

The pages should look almost identical to the original design, outside the content differences.
The pages use the exact same colors, for text, backgrounds, borders, etc, from the original design.


Basic color swatch: #272A31, #EC5242, #D3D3D3


The pages use the same fonts or simialr fonts from the original design.


Fonts used originally: Cocogoose, Lato


All page elements have similar spacing, borders, shadows, and paddings than the orignal design.


HTML code

The HTML uses semantic tags, sections use section, articles use article, the nav and footer use nav and footer.
There's at least 1, but no more than 1, h1 tag in each page.
The HTML markup is well structured and follows a clear hierarchy, with sections, and articles, each containing appropriated Hx headers and paragraphs.
The page is accessible, images contain alt attributes and there's not images or elements blocking the text in both screen sizes.
The logo contains a hyperlink to the homepage.


CSS code

Selector classes are used multiple times and rules are not repeated, the code is DRY.
The CSS styling is built as mobile-first, media queries are used to change the design from mobile to desktop, not the other way around.
Media queries are properly organized at the bottom of the stylesheet.
NOTE: Students can use CSS frameworks (Bootstrap, Tailwind CSS and any other CSS framework) for styling (if they want to).


Note: It is fine if the student has used Bootstrap or any other framework.


JavaScript code

Dynamically generated HTML

The section "Featured speakers" is generated dynamically with JavaScript.
The JavaScript code is within it's own .js file, not inside the HTML in scripttags.
The list of speakers used to generate the HTML is assigned to a variable as an array or object.
The list of speakers contain an speaker object for each that holds their specific data.


// You can use an array to store the objects
const speakerList = [ { name: "Speaker Name",... } ]
// It's ok to use an object too
const speakerList = { speaker1: { name: "Speaker Name",... } }

The object contains all the information needed to generate the dynamic element, at least the name, role, description, and image url about the speaker. For example:


{
  name: "Julia Leda", 
  role: "President of Young pirates of Europe", 
  description: "European ingetration, political democracy, and participation of ...", 
  photo: "image_url_here"
}

There is a function that iterates through the speakers list on window load and creates HTML elements with the data from each object. For example:


window.onload = () => {
  speakerList.map(speaker => {
    // code to build the new HTML element, it can use `createElement()` or just template strings(``).
    // it could attach the new element to a parent inside within the loop
    // or accumulate the generate elements in a variable and attach them all after it
  })

diarisdiakite replied:
Fixed @KayLemba, thanks for the review.

KayLemba reviewed (changes requested):
Hi @diarisdiakite  👋
Good job so far working on the project requirements!
There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight 🎉
No linter errors. ✔️
GitHub flow is followed. ✔️
Fixed most issue 

Required Changes ♻️
Check the comments under the review in the inline comments.

Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

diarisdiakite committed:
Add an about.js page to load about content based on the screen size

diarisdiakite committed:
Limit the speaker display to 2 on index.html mobile version page. fix page display issue. Delete comments in the code.

diarisdiakite commented:
In this PR, I:
- Fixed the display issue by reducing the h2 size
- Removed the partners' section and the footer from the mobile version of index.html
- Added the about.js to load content based on the window's width 
- Added only 2 speakers cards to the mobile version of index.html 
- removed the arrow from the mobile version of index.html

diarisdiakite committed:
Fix Eslint issue

Zuhaib042 commented on README.md:
```diff
@@ -0,0 +1,149 @@
+#### [Project-8](Details modal)
+#### [Project-9](Validate contact form)
+#### [Project-10](Preserve data in the browser)
+``
```
- [OPTIONAL] Kindly consider removing these lines as they are not concerned in this project. these milestones are related to portfolio-project.

Zuhaib042 commented on index.html:
```diff
@@ -1,13 +1,183 @@
+      
+    </div>
+    <!--ADD THE NAVBAR -->
+    <nav class="navbar navbar-expand-md navbar-light bg-light sticky-top" id="my-navbar">
+      
+    </nav>
```
- [OPTIONAL] Although this requirement is missed by the previous reviewer, I would highly recommend that you implement it. Kindly have a look at your design and the original design. 
In Original design in both Home and About pages, the hamburger menu has a transparent background and there is no nav background there. Neither the upper-level nav nor the lower level nav, so kindly consider removing these two navs in the mobile screen in both home and about pages.
And also consider to add Home page link in the navlinks in mobile screen for easy browsing.
your design
![fd](https://user-images.githubusercontent.com/96683843/235319882-0efc8a97-8980-4a52-ab5e-ab1edc149b92.jpg)

Original design

![as](https://user-images.githubusercontent.com/96683843/235319877-febfe775-b433-4767-9d95-e134c4f0cef6.jpg)

diarisdiakite replied:
The _Social_ section has been removed for all 2 pages, thank you for the comment. @zouhaib
I will work towards fixing the issue.

![image](https://user-images.githubusercontent.com/36234985/235321387-8103987e-364a-4edc-b9bd-57a5f1ee87d2.png)


![image](https://user-images.githubusercontent.com/36234985/235321245-22e3fe9a-5f3e-4cd2-830c-464192df924d.png)

Zuhaib042 reviewed (changes requested):
Hi @diarisdiakite 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
![](https://media.giphy.com/media/31Q36EHJ7C5ZoIMw79/giphy.gif)

 ## Highlights
No linter errors. ✔️
GitHub flow is followed. ✔️

 ## Required Changes ♻️
- [x]  As suggested by the previous reviewer @Qoosim , kindly record a video presenting the features of the project.
   - The video should contain a clear and objective description about the project in professional English.
   - The topic and purpose of the project should be made clear.
   - The video should contain a demo of the project, showing the pages in different screen 
   sizes.
   - The video should contain a highlight about a piece of the code you find interesting or 
   you're proud of making.
   - The video should have a maximum length of 5 minutes.
   Kindly do that.
Project Requirement
![qw](https://user-images.githubusercontent.com/96683843/235319481-10f5ce64-acd4-428c-9c30-7e32a82f198a.jpg)

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

diarisdiakite commented:
Please find the link to the loom Video. https://www.loom.com/share/f915379eeecf424883ed844b7a1568ea.
Please find as well the link to the first video: https://www.loom.com/share/7ea0fc378c374d87a9e7c9060381b260
I was adding them to the demo section of the project when submiting.
Any comment may help me improve my recordings for the future projects.
Thank you very much

diarisdiakite committed:
Add loom video link

diarisdiakite committed:
Update Readme.md

diarisdiakite commented:
In this milestone, I:
- add the link to the loom video: https://www.loom.com/share/f915379eeecf424883ed844b7a1568ea
- updated the readme
- fixed the socials display in the mobile versions.

Munsa1 reviewed (approved):
Hi @diarisdiakite, 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
